docs/BreakingChanges/052 WPF TextBox.Text can be out-of-sync with databinding.md

```diff
@@ -0,0 +1,24 @@
+## 52: WPF TextBox.Text can be out-of-sync with databinding.
```

Comment:
{
  "Id": "52",
  "Title": "WPF TextBox.Text can be out-of-sync with databinding.",
  "Details": "In some cases, the TextBox.Text property reflects a previous value of the databound property value if the property is modified during a databinding write operation.",
  "Suggestion": "This should have no negative impact. However, you can restore the previous behavior by setting the [FrameworkCompatibilityPreferences.KeepTextBoxDisplaySynchronizedWithTextProperty](https://msdn.microsoft.com/en-us/library/system.windows.frameworkcompatibilitypreferences.keeptextboxdisplaysynchronizedwithtextproperty(v=vs.110).aspx) property to false.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#wpf",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Windows.Controls.TextBox.get_Text",
    "M:System.Windows.Controls.TextBox.set_Text(System.String)",
    "P:System.Windows.Controls.TextBox.Text"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": true,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Windows.Controls.TextBox.get_Text
- [x] M:System.Windows.Controls.TextBox.set_Text(System.String)
- [x] P:System.Windows.Controls.TextBox.Text